exercises/variable-length-quantity/tests/variable-length-quantity.rs

```diff
@@ -0,0 +1,89 @@
+extern crate variable_length_quantity as vlq;
+
+#[test]
+fn to_single_byte() {
+    assert_eq!(&[0x40], vlq::to_bytes(&[0x40]).as_slice());
+    assert_eq!(&[0x7f], vlq::to_bytes(&[0x7f]).as_slice());
+}
+
+#[test]
+#[ignore]
+fn to_single_byte_zero() {
+    assert_eq!(&[0x00], vlq::to_bytes(&[0x00]).as_slice());
+}
+
+#[test]
+#[ignore]
+fn to_double_byte() {
+    assert_eq!(&[0x81, 0x00], vlq::to_bytes(&[0x80]).as_slice());
```

Comment:
Similar to my earlier comment, are there edge cases that these additional tests are capturing? I've found that tests that focus on one thing work best. So if each of these asserts is testing a unique implementation detail, then I'd make this 4 tests.

But if getting the first assert to pass will also make the remaining 3 asserts pass, then I'd get rid of the additional asserts.